### PR TITLE
[zh]Add kubelet-credentialprovider.v1alpha1.md chinese version

### PR DESCRIPTION
**Pull request by zhangxyjlu**
fix: #34177
Translate Chinese version for reference/config-api/kubelet-credentialprovider.v1alpha1.md

### CODE REVIEW

**Comment by Sea-n** [comment]
We have reviewed #34176.
- #34176

This PR is for `v1alpha1` version, the only difference is `v1beta1` and `v1alpha1`.
/lgtm

File: content/zh/docs/reference/config-api/kubelet-credentialprovider.v1alpha1.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
+---
+title: Kubelet 凭据提供程序（CredentialProvider）（v1alpha1）
```
Comment (tengqm) [inline content/zh/docs/reference/config-api/kubelet-credentialprovider.v1alpha1.md:2]: 这个类别名称不要翻译。

```suggestion
title: Kubelet CredentialProvider（v1alpha1）
```

File: content/zh/docs/reference/config-api/kubelet-credentialprovider.v1alpha1.md [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+
+## `CredentialProviderRequest`     {#credentialprovider-kubelet-k8s-io-v1alpha1-CredentialProviderRequest}
+
+<!--
+CredentialProviderRequest includes the image that the kubelet requires authentication for.
+Kubelet will pass this request object to the plugin via stdin. In general, plugins should
+prefer responding with the same apiVersion they were sent.
+-->
+<p>
+CredentialProviderRequest 包括 kubelet 需要进行身份验证的镜像。
```
Comment (tengqm) [inline content/zh/docs/reference/config-api/kubelet-credentialprovider.v1alpha1.md:29]:
```suggestion
CredentialProviderRequest 包含 kubelet 需要进行身份验证的镜像。
```

**Review by Sea-n — COMMENTED** [review]
Thanks for fixing `v1beta1` as well, this PR looks good to me! 🎉
/lgtm

**Pull request merged**